WTWSMS/history/provinces/788 - Farama.txt

```diff
@@ -16,11 +16,26 @@ b_seyan = city
 #b_mustabiq = castle
 
 # Misc
-culture = greek
-religion = docetic 
+culture = coptic
+religion = sethian
```

Comment:
The first source refers to 395 though. Even if we have discussed a possibility of adding *something* to reflect a tendency of divergence in desert areas, this account of there being Sethians is insufficient to justify their inclusion as a majority in any province.